spec/models/vocab_sheet_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe 'VocabSheet', type: :model do
+  describe '#clear_vocab_sheet' do
+    it 'only deletes a vocab sheet older than 31 days' do
+      FactoryBot.create_list(:vocab_sheet, 5)
+      vocab_sheet = FactoryBot.create(:vocab_sheet, created_at: 1.year.ago, updated_at: 1.year.ago)
```

Comment:
I would make the old record as close to 31 days as possible to test the edge case - like: `31.days.ago - 1.minute` or `32.days.ago`